tests/foreman/api/test_user.py

```diff
@@ -6,19 +6,19 @@
 """
 import ddt
 from fauxfactory import gen_string
+from nailgun import client
 from random import randint
 from requests.exceptions import HTTPError
-from robottelo import entities
+from robottelo.common.helpers import get_server_credentials
 from robottelo.common import decorators
+from robottelo import entities
```

Comment:
Sure. Oddly enough, this is the ordering produced by `sort`. I didn't see an easy way to change this behaviour after flipping through the man page. I'll let you know if I ever dig further into this and find a nice solution.